ghcjs-src/Miso/Subscription/Geolocation.hs

```diff
@@ -0,0 +1,83 @@
+{-# LANGUAGE RecordWildCards   #-}
+{-# LANGUAGE OverloadedStrings #-}
+-----------------------------------------------------------------------------
+-- |
+-- Module      :  Miso.Subscription.Geolocation
+-- Copyright   :  (C) 2016-2017 David M. Johnson
+-- License     :  BSD3-style (see the file LICENSE)
+-- Maintainer  :  David M. Johnson <djohnson.m@gmail.com>
+-- Stability   :  experimental
+-- Portability :  non-portable
+----------------------------------------------------------------------------
+module Miso.Subscription.Geolocation
+  ( geoSub
+  , defGeoOptions
+  , GeoOptions (..)
+  , Geo (..)
+  ) where
+
+import GHCJS.Foreign.Callback
+import GHCJS.Marshal
+import GHCJS.Types
+import JavaScript.Object
+import JavaScript.Object.Internal
+
+import Miso.Html.Internal         ( Sub )
+import Miso.String
+
+data GeoOptions = GeoOptions
```

Comment:
The haddocks here should link to the docs for [PositionOptions](https://developer.mozilla.org/en-US/docs/Web/API/PositionOptions). Personally I would also make the fields strict but I don’t have a particular strong opinion in this case.